substrait_consumer/consumers/acero_consumer.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import string
+from pathlib import Path
+from typing import Iterable
+
+import pyarrow as pa
+import pyarrow.parquet as pq
+import pyarrow.substrait as substrait
+
+from substrait_consumer.common import SubstraitUtils
+
+from .consumer import COLUMN_A, COLUMN_B, COLUMN_C, COLUMN_D, Consumer
+
+
+class AceroConsumer(Consumer):
+    """
+    Adapts the Acero Substrait consumer to the test framework.
+    """
+
+    def __init__(self):
+        self.tables = {}
+        self.table_provider = lambda names, schema: self.tables[names[0].lower()]
+
+    def setup(self, db_connection, created_tables, file_names: Iterable[str]):
+        if len(file_names) > 0:
+            parquet_file_paths = SubstraitUtils.get_full_path(file_names)
+            for file_name, file_path in zip(file_names, parquet_file_paths):
+                table_name = Path(file_name).stem
```

Comment:
Eventually should the logic to create a table be another abstract method on the base class?

Reply:
Makes sense...i'll create an issue to track this.